ingen/data_source/file_source.py

```diff
@@ -31,6 +31,8 @@ def __init__(self, source, params_map):
             self._src = source
         else:
             self._src = self.format_file_path(source, params_map)
+        if not source.get('file_encoding'):
```

Comment:
@ChillarAnand I was thinking to remove this default setting here and add defaults in pd.read_csv calls in file_reader and xml_reader. Like `encoding=src.get('file_encoding', 'utf-8')`. The benefit is, if we set default closer to where it is being used it'd be easier for the reader to understand that utf-8 is the default encoding. Let me know your thoughts?